entity-data-service-rx-client/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+  `java-library`
+  id("org.hypertrace.publish-plugin")
+  id("org.hypertrace.jacoco-report-plugin")
```

Comment:
add jacoco?

Reply:
whoops - the ETS client that I snagged this from was missing too, will update both.